tests/postgres_tests/test_array.py

```diff
@@ -117,6 +117,10 @@ def test_model_set_on_base_field(self):
         self.assertEqual(field.model, IntegerArrayModel)
         self.assertEqual(field.base_field.model, IntegerArrayModel)
 
+    def test_choices(self):
+        instance = ChoiceIntegerArrayModel(field=[1, 2])
```

Comment:
Your test doesn't need to hit the database so you can inline the model in the test, decorate the test with `@isolate_apps`, and remove the model from the migration.